### DISCUSSION TITLE
Github education ID proof verification has serious technical glitch

### DISCUSSION THREAD

**Discussion by sleepingcat4** [community/community#123970] [GitHub Education]
### Hi everyone,

I was trying to upload my academic proof to receive Github student benefits since high school ID will be expiring soon and I am now in University. That's why, I wanted to verify my university ID to receive student benefits. I have verified my university emails and everything. 

Unfortunately, when I am asked to upload academic proof, I only receive an option "Take a Photo" using the front camera both on my laptop and iPhone and Android phone as well. I tried using the back camera option it does not work on both my iPhone and Android. 

I am seriously confused and frustrated, how Github can have such a serious bug and also why I need take a photo in the first place rather than uploading a (.png/.jpeg) file to the Github verification method and they verify my status. Why it needs to be so complicated and frustrating. It's literally 2024 and this super bad and poor UI choice and design is still tolerated. It's a bloody shame. 

Cherry on top. Now, you can't open tickets to get the verification done manually. Like I am literally confused they want me to do. Buy a 4K monitor and then buy a HD Camera attach to my laptop and then take a snapshot? Please, someone help me out.

**Comment by oSam2** [comment]
I'm experiencing the same issue.

**Reply by sleepingcat4** [reply]
It's super frustrating!

**Comment by sleepingcat4** [comment]
I will include further pain-points:

1. I am residing in a different country now as I will be going back to United states in August. That's why, my university location and current location are a mismatch. 
2. The academic proof, I have does not have a date specified. Since when I downloaded it from my university's website (.pdf) did not include the original date when it was issed. 
3. Why I need to provide my full name? Please specify. I don't use my exact legal full name while using GitHub. Because in academic settings, I use a different name from my full legal name that's reflected on my legal documents. 

I want to emphasize, these are ridiculous askings from someone like Github. It's 2024 not 1994. Literally, just using my university email with (.edu) domain should be enough to verify my status. Almost everywhere I redeem student benefits including other university and professors, the (.edu) email is enough for them to affirm my student status. 

For kind information, I still couln't upload an image. Either fix the problem with your website otherwise bring back option for manual verification. this current system is a JOKE.

**Comment by darrenphilip** [comment]
I am having the same issue aswell its quite a pain
